src/schema/schema_manager.py

```diff
@@ -507,6 +516,25 @@ def remove_transient_and_none_values(merged_dict, normalized_entity_type):
     return filtered_dict
 
 
+"""
+Update entity via HTTP call to entity-api
+This is useful when a change in one entity necessitates changes in another, while allowing the normal triggers
+and validators to execute
+
+Parameters
+----------
+uuid : string
+    The uuid of the entity being updated
+data : dict
+    A dict representation of the json_data_dict for the updated entity
+token : string
+    The globus groups token
+"""
+def update_entity(uuid, data, token):
+    url = _entity_api_url + SchemaConstants.ENTITY_API_UPDATE_ENDPOINT +  '/' + uuid
+    header = _create_request_headers(token)
+    header[schemaConstants.HUBMAP_APP_HEADER] = INGEST_API_APP
```

Comment:
I didn't test, at a quick glance `INGEST_API_APP` should be referenced using `schemaConstants.`.